packages/neuron-wallet/src/database/address/address-dao.ts

```diff
@@ -113,25 +141,41 @@ export default class AddressDao {
         && value.version === version
         && value.addressType == AddressType.Receiving
         && value.txCount === 0
+        && !value.usedByAnyoneCanPay
```

Comment:
Isn't `value.txCount === 0` already covering `!value.usedByAnyoneCanPay`?

@classicalliu

Reply:
My understanding is when an address is used to send an ACP transaction, then the `txCount` will be incremented for the corresponding address.

Reply:
`txCount` only covers transactions which uses default lock.